src/components/DigitalCredentials/CredentialsStepper.vue

```diff
@@ -121,7 +131,7 @@
             class="pt-3 px-3"
           >
             <v-card-text
-              v-if="!issuedCredential"
+              v-if="!issuedCredential?.isIssued"
```

Comment:
Caution: this will evaluate to True if issueCredential is falsy. Maybe compare with false or rework the statement?

PS It used to be that optional chaining didn't work in the template (since here it's JS code, not TS).

Reply:
Hmm, I think this is covered by the v-else above it.

Reply:
Maybe! So then you don't need the optional chaining here. And I hope nobody changes the logic in the template without catching this.

I also see the same issue on line 105.